src/fswAlgorithms/effectorInterfaces/solarArrayReference/solarArrayReference.h

```diff
@@ -25,29 +25,43 @@
 #include "cMsgCInterface/NavAttMsg_C.h"
 #include "cMsgCInterface/AttRefMsg_C.h"
 #include "cMsgCInterface/HingedRigidBodyMsg_C.h"
+#include "cMsgCInterface/VehicleConfigMsg_C.h"
+#include "cMsgCInterface/RWSpeedMsg_C.h"
+#include "cMsgCInterface/RWArrayConfigMsg_C.h"
 
 
-enum attitudeFrame{
+typedef enum attitudeFrame{
     referenceFrame = 0,
     bodyFrame = 1
-};
+} AttitudeFrame;
+
+typedef enum pointingMode{
+    powerGeneration = 0,
+    momentumDumping  = 1
+} PointingMode;
 
 /*! @brief Top level structure for the sub-module routines. */
 typedef struct {
 
     /* declare these user-defined quantities */
     double a1Hat_B[3];              //!< solar array drive axis in body frame coordinates
     double a2Hat_B[3];              //!< solar array surface normal at zero rotation
-    int attitudeFrame;              //!< flag = 1: compute theta reference based on current attitude instead of attitude reference
+    AttitudeFrame attitudeFrame;    //!< flag = 1: compute theta reference based on current attitude instead of attitude reference
```

Comment:
This will not cause backwards compatibility issues because the typedef enum structure is still essentially an integer. 

I have updated the comment.